test/unit/integrations/poli_pay_module_test.rb

```diff
@@ -0,0 +1,114 @@
+require 'test_helper'
+
+class PoliPayModuleTest < Test::Unit::TestCase
+  include ActionViewHelperTestHelper
+  include OffsitePayments::Integrations
+
+  def setup
+    super
+    @options = fixtures(:poli_pay)
+  end
+
+  def transaction_options
+    {
+      amount:       157.0,
+      currency:     'AUD',
+      return_url:   "http://www.sealink.com.au/polipay_return",
+      homepage_url: 'http://www.sealink.com.au',
+      credential2:   @options[:password]
+    }
+  end
+
+  def test_bad_password
+    options = transaction_options
+    options[:credential2] = 'wrong'
+    helper = PoliPay::Helper.new('22FART', @options[:login], options)
```

Comment:
Good point, resolved in additional commit

On 12/01/16 13:11, Alessandro Berardi wrote:
>
> In test/unit/integrations/poli_pay_module_test.rb 
> <https://github.com/sealink/offsite_payments/pull/1#discussion_r49409946>:
>
> > +  end
> > +
> > +  def transaction_options
> > +    {
> > +      amount:       157.0,
> > +      currency:     'AUD',
> > +      return_url:   "http://www.sealink.com.au/polipay_return",
> > +      homepage_url: 'http://www.sealink.com.au',
> > +      credential2:   @options[:password]
> > +    }
> > +  end
> > +
> > +  def test_bad_password
> > +    options = transaction_options
> > +    options[:credential2] = 'wrong'
> > +    helper = PoliPay::Helper.new('22FART', @options[:login], options)
>
> I wouldn't use that reference in a public gem @mnoack 
> <https://github.com/mnoack>
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/sealink/offsite_payments/pull/1/files#r49409946>.
>
>
> ______________________________________________________________________
> This email has been scanned by the Symantec Email Security.cloud service.
> For more information please visit http://www.symanteccloud.com
> ______________________________________________________________________